src/components/InlineLoading/_index.scss

```diff
@@ -4,7 +4,9 @@
 /// @copyright IBM Security 2019
 ////
 
+@import '../../globals/variables/index';
```

Comment:
Can you explicitly import `@carbon/themes/scss/mixins` in files where we are using them?

Reply:
Will import:

```scss
@import '@carbon/themes/scss/tokens';
```